api/openapi/websocket.yml

```diff
@@ -4,7 +4,7 @@ info:
   description: WebSocket API for sending messages through communication channels.
   version: "1.0.0"
 paths:
-  /channels/{id}/messages:
```

Comment:
Fix this swagger file. You can use a tool [like](https://editor.swagger.io/) this to cross check the edited swagger files